venues/aclweb.org/ACL/post_desk_reject_invitations.py

```diff
@@ -0,0 +1,96 @@
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+sac_name_dictionary = {
+    'Ethics in NLP': 'Ethics_NLP',
+    'Linguistic theories, Cognitive Modeling and Psycholinguistics': 'LCMP',
+    'Linguistic Theories, Cognitive Modeling and Psycholinguistics': 'LCMP',
+    'Machine Learning for NLP': 'Machine_Learning_NLP',
+    'Phonology, Morphology and Word Segmentation': 'Phonology_Morphology_Word_Segmentation',
+    'Resources and Evaluation': 'Resources_Evaluation',
+    'Semantics: Lexical': 'Semantics_Lexical',
+    'Semantics: Sentence level, Textual Inference and Other areas': 'Semantics_STO',
+    'Syntax: Tagging, Chunking and Parsing': 'Syntax_TCP',
+    'Information Extraction': 'Information_Extraction',
+    'Computational Social Science and Cultural Analytics': 'CSSCA',
+    'Information Retrieval and Text Mining': 'Info_Retrieval_Text_Mining',
+    'Interpretability and Analysis of Models for NLP': 'IAM_for_NLP',
+    'Machine Translation and Multilinguality': 'Machine_Translation_Multilinguality',
+    'NLP Applications': 'NLP_Applications',
+    'Question Answering': 'Question_Answering',
+    'Dialogue and Interactive Systems': 'Dialogue_and_Interactive_Systems',
+    'Discourse and Pragmatics': 'Discourse_and_Pragmatics',
+    'Generation': 'Generation',
+    'Language Grounding to Vision, Robotics, and Beyond': 'LGVRB',
+    'Sentiment Analysis, Stylistic Analysis, and Argument Mining': 'SASAAM',
+    'Speech and Multimodality': 'Speech_and_Multimodality',
+    'Summarization': 'Summarization',
+    'Special Theme on Language Diversity: From Low Resource to Endangered Languages': 'Special_Theme'
+    }
+
+
+
+# Get all ACL 2022 Conference blind submissions 
+acl_blind_submissions = list(openreview.tools.iterget_notes(client, invitation = 'aclweb.org/ACL/2022/Conference/-/Blind_Submission'))
+# For each blind submission, set the readers to the SAC track group 
+for acl_blind_submission in acl_blind_submissions:
+    acl_submission = client.get_note(acl_blind_submission.original)
+    desk_reject = client.post_invitation(openreview.Invitation(
+        id = f"aclweb.org/ACL/2022/Conference/Paper{acl_blind_submission.number}/-/Desk_Reject",
+        invitees = ["aclweb.org/ACL/2022/Conference/Program_Chairs","OpenReview.net/Support"],
+        readers = ["everyone"],
+        writers = ["aclweb.org/ACL/2022/Conference"],
+        signatures = ["~Super_User1"],
+        reply = {
+            "forum": acl_blind_submission.forum,
+            "replyto": acl_blind_submission.forum,
+            "readers": {
+                "values": [
+                    "aclweb.org/ACL/2022/Conference",
+                    f'aclweb.org/ACL/2022/Conference/{sac_name_dictionary[acl_submission.content["track"]]}/Senior_Area_Chairs'
```

Comment:
if we are adding track SACs as readers of the desk-reject comment, then we should also send them an email. I am not sure when the PCs will desk-reject papers (before or after SACs are added to readers)